var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -181,6 +181,9 @@ def setup_build_environment(self, env):
         # https://bugzilla.redhat.com/show_bug.cgi?id=1795817
         if self.spec.satisfies('%gcc@10:'):
             env.set('FFLAGS', '-fallow-argument-mismatch')
+        # Same fix but for max - avoids issue #17934
```

Comment:
```suggestion
        # Same fix but for macOS - avoids issue #17934
```
Is this what you meant?